src/containers/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from "react";
+import FilterBar from "../FilterBar/FilterBar";
+import MovieList from "../MovieList/MovieList";
+
+import styles from "./Dashboard.module.scss";
+import GENRES from "../../core/constants/GENRES";
+import SORTOPTIONS from "../../core/constants/SORTOPTIONS";
+import getMoviesMocked from "../../core/api/getMoviesMocked";
+
+const Dashboard = (): JSX.Element => {
+  const [moviesState, setMoviesState] = useState([]);
+
+  const fetchMovies = async (): Promise<void> => {
```

Comment:
It would make sense to move "fetchMovies" to "core/api".
"fetchMovies" looks like a good abstraction on top of real/mocked api, so that components don't really know/care about the data source.
Therefore, "setMoviesState" also should be moved, e.g to the "useEffect" hook.